src/components/ambassadorCard/AmbassadorCard.tsx

```diff
@@ -30,142 +32,156 @@ export interface AmbassadorCardProps {
   ambassador: AmbassadorType;
   onClose?: () => void;
   className?: string;
+  disableCardEffects?: boolean;
 }
 
 export default function AmbassadorCard(props: AmbassadorCardProps) {
-  const { ambassadorKey, ambassador, onClose, className } = props;
+  const { ambassadorKey, ambassador, onClose, className, disableCardEffects } =
+    props;
   const images = getAmbassadorImages(ambassadorKey);
   const mod =
     window?.Twitch?.ext?.viewer?.role === "broadcaster" ||
     window?.Twitch?.ext?.viewer?.role === "moderator";
+  const glareOpacity = disableCardEffects ? 0.0 : 0.5;
 
   return (
-    <div
-      className={classes(
-        styles.ambassadorCard,
-        className,
-        ambassador.birth && isBirthday(ambassador.birth) && styles.birthday,
-      )}
+    <Tilt
+      tiltEnable={!disableCardEffects}
+      glareEnable={!disableCardEffects}
+      glareMaxOpacity={glareOpacity}
+      glareBorderRadius="1rem"
+      glarePosition="bottom"
+      scale={1.0}
+      perspective={5000}
```

Comment:
```suggestion
      perspective={7500}
      tiltReverse
```
From playing around with the settings locally, this felt nicer to me?

Reply:
Agree! Change made